src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -127,6 +127,101 @@ You can use the `required` prop to indicate if the field is mandatory.
   </Story>
 </Preview>
 
+### With object as value:
+
```

Comment:
We should also add a description here explaining why you would want to use an option.